src/gui/transferlistmodel.h

```diff
@@ -76,11 +76,14 @@ class TransferListModel final : public QAbstractListModel
         TR_AMOUNT_LEFT,
         TR_TIME_ELAPSED,
         TR_SAVE_PATH,
+        TR_DOWNLOAD_PATH,
```

Comment:
IIRC adding it to the middle of the list will mess up existing config of the columns. So move it to the bottom of the list.